icechunk-python/Cargo.toml

```diff
@@ -37,6 +37,7 @@ serde_json = "1.0.137"
 async-trait = "0.1.85"
 typetag = "0.2.19"
 serde = { version = "1.0.217", features = ["derive", "rc"] }
+miette = { version = "7.5.0", features = ["fancy"] }
```

Comment:
this is used purely to generate the exception message string, with a nice report of all the chain of exception causes, some colors, etc